test/requests/git_hub/client_test.rb

```diff
@@ -116,28 +116,30 @@ def test_successful_commits_for
     commit = response.parsed_response.first
 
     assert response.headers['link'].nil?
-    assert response.parsed_response.class == Array
-    assert response.parsed_response.size == 1
-    assert commit['sha'] == '3d4ff7164a33fa1c6006d34942872d1c22594b99'
-    assert commit['html_url'] == 'https://github.com/OperationCode/operationcode/commit/3d4ff7164a33fa1c6006d34942872d1c22594b99'
-    assert commit['commit']['message'] == "Revert \"waffle.io Badge (#752)\"\n\nThis reverts commit ac38898ddeae5ddd17cdb6addba1b52fc6da923a."
-    assert commit['commit']['committer']['date'] == '2017-07-02T20:31:56Z'
-
-    assert commit['author']['login'] == 'hollomancer'
-    assert commit['author']['avatar_url'] == 'https://avatars3.githubusercontent.com/u/9288648?v=4'
-    assert commit['author']['url'] == 'https://api.github.com/users/hollomancer'
-    assert commit['author']['html_url'] == 'https://github.com/hollomancer'
-    assert commit['author']['id'] == 9288648
+    assert_equal Array, response.parsed_response.class
+    assert_equal 1, response.parsed_response.size
+    assert_equal '3d4ff7164a33fa1c6006d34942872d1c22594b99', commit['sha']
+    assert_equal 'https://github.com/OperationCode/operationcode/commit/3d4ff7164a33fa1c6006d34942872d1c22594b99', commit['html_url']
+    assert_equal "Revert \"waffle.io Badge (#752)\"\n\nThis reverts commit ac38898ddeae5ddd17cdb6addba1b52fc6da923a.", commit['commit']['message']
+    assert_equal '2017-07-02T20:31:56Z', commit['commit']['committer']['date']
+
+    assert_equal 'hollomancer', commit['author']['login']
+    assert_equal 'https://avatars3.githubusercontent.com/u/9288648?v=4', commit['author']['avatar_url']
+    assert_equal 'https://api.github.com/users/hollomancer', commit['author']['url']
+    assert_equal 'https://github.com/hollomancer', commit['author']['html_url']
+    assert_equal 9288648, commit['author']['id']
   end
 
   def test_rate_limit_presence
     response = VCR.use_cassette('git_hub/commits/successful') do
       @client.commits_for(@oc, @pr_number, 1)
     end
 
-    assert response.headers['X-RateLimit-Remaining'] == '57'
-    assert response.headers['X-RateLimit-Limit'] == '60'
-    assert response.headers['X-RateLimit-Reset'] == '1503866476'
-    assert @client.send(:reset_time, response) == '8/27/2017 at 8:41:16pm UTC'
+    reset_time = Time.at(1503866476).strftime '%-m/%d/%Y at%l:%M:%S%P %Z'
```

Comment:
How does this break test runs?

Reply:
When running locally it uses the local time zone when generating the timestamp which doesn’t match the expected value which is UTC.

Reply:
So I remember that sometimes the rate limiting test messes things up. We should probably have a way of testing rate limiting. 

Why is this done instead of some other way?

Reply:
I can pull in something like timecop to manage the expected timezone